iio.h

```diff
@@ -253,6 +254,42 @@ __api __check_ret __pure const char * iio_context_info_get_uri(
 		const struct iio_context_info *info);
 
 
+/** @brief Create a scan block
+ * @param backend A NULL-terminated string containing the backend to use for
+ * scanning. If NULL, all the available backends are used.
+ * @param flags Unused for now. Set to 0.
+ * @return on success, a pointer to a iio_scan_block structure
+ * @return On failure, NULL is returned and errno is set appropriately */
+__api struct iio_scan_block * iio_create_scan_block(
```

Comment:
Can we add a 
```
Introduced in version 0.20. */
```

in the comment for all?

Reply:
added